.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+C:37:"PHPUnit\Runner\DefaultTestResultCache":759:{a:2:{s:7:"defects";a:5:{s:51:"Webleit\ZohoCrmApi\Test\ApiTest::canGenerateAuthUrl";i:4;s:47:"Webleit\ZohoCrmApi\Test\ApiTest::hasAccessToken";i:1;s:55:"Webleit\ZohoCrmApi\Test\ApiTest::canGenerateAccessToken";i:1;s:51:"Webleit\ZohoCrmApi\Test\ApiTest::canParseGrantToken";i:4;s:54:"Webleit\ZohoCrmApi\Test\ApiTest::canRefreshAccessToken";i:4;}s:5:"times";a:6:{s:51:"Webleit\ZohoCrmApi\Test\ApiTest::canGenerateAuthUrl";d:0.002;s:47:"Webleit\ZohoCrmApi\Test\ApiTest::hasAccessToken";d:0;s:55:"Webleit\ZohoCrmApi\Test\ApiTest::canGenerateAccessToken";d:0;s:51:"Webleit\ZohoCrmApi\Test\ApiTest::canParseGrantToken";d:0.672;s:54:"Webleit\ZohoCrmApi\Test\ApiTest::canRefreshAccessToken";d:0.566;s:52:"Webleit\ZohoCrmApi\Test\ApiTest::canCacheAccessToken";d:0.56;}}}
```

Comment:
Did you mean to commit this?

Reply:
No, i'll add it to the gitignore, thanks